js-sdk/providing_static_data.mdx

```diff
@@ -141,8 +141,21 @@ const tolgee = Tolgee()
 
 `fallbackOnFail` option will tell Tolgee to look for data in the next source, so it will fetch your local data. **Order of plugins is important here, they are used in the same order as they were added.**
 
-:::info Plugins are used in the same order as they were added.
+### Fallback logic
+
+Plugins are used in the same order as they were added.
 Tolgee first checks if data aren't already in cache (data passed directly to `staticData`) then all the plugins and then dynamic data from `staticData` field.
+
+:::warning Don't pass data directly
+Passing data directly (e.g. `en: enObject`), results in data being used as a first option - without further attempts to load data.
```

Comment:
I would provide this example also this way so it's clear how we are passing the object.

```ts
staticData: {
  en: enObject
  ...
}
```

Reply:
Ok